.docker/alpine/lean/Dockerfile

```
@@ -0,0 +1,75 @@
+# This is an alternative Dockerfile for `leanprover/lean`,
+# based on `alpine`, and installs `elan` and `leanproject`.
+# See also the image `leanprover/mathlib` which contains a copy of mathlib.
+
+# `elan` doesn't include a release for `x86_64-unknown-linux-musl`,
+# so we roll our own
+FROM clux/muslrust as elan-builder
+
+RUN git clone https://github.com/Kha/elan.git .
+RUN cargo build
```

```suggestion
RUN cargo build --release
```

And then also replace `debug` by `release` in the COPY below.  This is probably part of the large size.

---

Perfect! We're now down to a 75mb `alpine` container.